ejb3/src/main/java/org/jboss/as/ejb3/remote/LocalEjbReceiver.java

```diff
@@ -697,7 +697,7 @@ private void removeClusterNodes(final EJBClientContext ejbClientContext, final S
         for (Map.Entry<String, List<ClientMapping>> entry : removedNodes.entrySet()) {
             final String removedNodeName = entry.getKey();
             // if the current node is being removed, then let the local receiver handle it
-            if (LocalEjbReceiver.this.getNodeName().equals(removedNodeName)) {
+            if (LocalEjbReceiver.this.getNodeName().equals(removedNodeName) && clusterContext != null) {
```

Comment:
This is most likely treating the symptoms rather than finding and understanding why and what closed the context prior to this. I think we need to understand that first and the fix would likely look different.